### PR TITLE
Updating record label name fixed

### PR DESCRIPTION
**Pull request by maheshkkumar**
- Added exception handling to addLabel function in feedback.js

### CODE REVIEW

File: feedback-front/static/js/feedback.js [hunk 1]
```diff
@@ -43,12 +43,40 @@ var FeedBackClient = function(name) {
...
+                  document.write("Internal Server Error [500].\n Redirecting to the page ......");
+                  window.setTimeout(function(){location.reload()},900);
+              } else if (exception === 'parsererror') {
+                  document.write("Requested JSON parse failed.\n Redirecting to the page ......");
+                  window.setTimeout(function(){location.reload()},900);
+              } else if (exception === 'timeout') {
+                  document.write("Time out error.\n Redirecting to the page ......");
+                  window.setTimeout(function(){location.reload()},900);
+              } else if (exception === 'abort') {
+                  document.write("Ajax request aborted.\n Redirecting to the page ......");
```
Comment (samarthbhargav) [inline feedback-front/static/js/feedback.js]: document.write?? Don't use this dude.

File: feedback-front/static/js/feedback.js [hunk 2]
```diff
@@ -41,14 +41,36 @@ var FeedBackClient = function(name) {
...
+            $('#labelNameStatus').html("<div style='color:green'>Label Name was added successfully.</div>");
+            window.setTimeout(function(){location.reload()}, 1500);
           }
-        }).fail(function (jqxhr) {
+        }).fail(function (jqxhr, exception) {
             // Display failure message in UI
-            console.log(" Adding new label value failed!");
+                console.log(" Adding new label value failed!");
+              if (jqXHR.status === 0) {
+                  $('#labelNameStatus').html("<div style='color:red'>Not connecting, verify your network</div>");
```
Comment (samarthbhargav) [inline feedback-front/static/js/feedback.js]: What are you doing here?
do this: define a `var msg` and set it to whatever is the content of the div - for ex `Not connecting, ...` etc. 
At the end of the if-else-if ladder do a `$('#labelNameStatus').html("<div style='color:red'>" + msg +"</div>");`

Comment (samarthbhargav) [inline feedback-front/static/js/feedback.js]: Don't-Repeat-Yourself or `DRY` is a very fundamental principle :+1:

File: feedback-front/static/js/feedback.js [hunk 3]
```diff
@@ -117,6 +146,8 @@ var FeedBackClient = function(name) {
           data: JSON.stringify(json),
           success: function(data) {
             // TODO:Mahesh Display success message in UI
+            $('#datasetStatus').html("<div style='color:green'>Dataset was added successfully.</div>");
```
Comment (samarthbhargav) [inline feedback-front/static/js/feedback.js:149]: Same here

**Comment by samarthbhargav** [comment]
@maheshkkumar  everything else looks good - please address the comments above and I'll merge it.

**Comment by maheshkkumar** [comment]
- I have implemented two functions one for success and another for failure, which will accept a string as a parameter and will display that particular string with it's corresponding color, either green for success  or red for failure.

**Pull request merged**